### ISSUE TITLE
Incorrect syntax error after upgrade to 1.6.5

### ISSUE THREAD

**Issue by aghArdeshir** [styled-components/vscode-styled-components#317]
**Describe the bug (including copyable syntax)**
If you're planning on posting a screenshot, please paste the accompanying code here.

Everything was good, today I upgraded to `1.6.5`, and I received errors for this code:
```ts
export const fadeIn = keyframes`
  0% {
    opacity: 0;
  }

  100% {
    opacity: 1;
  }
`;
```
I reverted back to 1 version before (published almost 1 month ago), and it was good again.

**Screenshot**
Screenshots are alway useful!
![Screenshot from 2021-07-26 12-41-58](https://user-images.githubusercontent.com/5755214/126960540-01a289e8-f25b-45dd-9e3d-20526e47fd0c.png)
![Screenshot from 2021-07-26 12-42-00](https://user-images.githubusercontent.com/5755214/126960548-6fad293c-7d4f-4f00-aa54-0d8abd382e2d.png)

**To Reproduce**
Steps to reproduce the issue, or JavaScript code that causes this failure.
- Have VSCode-Insiders 1.59.0-insider
- Install vscode-styled-components v1.6.5
- Create a `.ts` file including the code snippet above

**Expected behavior**
Explain what you expected to happen, and what is happening instead.

**Build environment (please complete the following information):**

- OS: [e.g. Fedora Linux] Ubuntu 20.04
- VSCode Version: [e.g. 1.49] 1.59.0-insider
- Extension Version [e.g. 1.0.0] 1.6.5

**Additional context**

**Comment by jasonwilliams** [comment]
Looks like the upgrade to https://github.com/microsoft/typescript-styled-plugin v0.18 caused the keyframes issue. Hopefully today i can either fix it or roll back to the previous version.

Not sure why that issue is back, it was fixed, looks like there's another bug somewhere

**Comment by aghArdeshir** [comment]
Thank you for taking care of it :+1:

**Comment by jasonwilliams** [comment]
I've tested directly against the head of typescript-styled-plugin and it works, so it looks like an issue on their side (as their published version looks to be behind).

Raised here:
https://github.com/microsoft/typescript-styled-plugin/issues/143

I've put out a new release (1.6.6) which disables intelliesense in keyframes until the upstream issue is fixed

**Comment by jasonwilliams** [comment]
Now that Typescript styled plugin [is updated](https://github.com/microsoft/typescript-styled-plugin/issues/143#issuecomment-893913699) it should just be a case of updating to the latest version.

**Comment by jasonwilliams** [comment]
@aghArdeshir is this now fixed on 1.7.2?

**Comment by aghArdeshir** [comment]
Hi @jasonwilliams 
I don't have the exact same reproduction environment as the time I created this issue, but with this configuration:
```
Version: 1.64.0-insider
TypeScript: 4.5.3
vscode-styled-components: 1.7.0
```
The issue seems to be resolved. 

Note that my `vscode-styled-components` is `v1.7.0` and not `v1.7.2` that you mentioned. It seems to be at its latest with no update available. 

![image](https://user-images.githubusercontent.com/5755214/145779960-fa81a65d-8f35-419a-9d0a-c457aeb370b2.png)

**Comment by jasonwilliams** [comment]
@aghArdeshir see top of readme for this repo

**Comment by aghArdeshir** [comment]
Thanks @jasonwilliams 
Yep! I updated it using the new extension source and Yes! Still everything works

And thanks for the update :+1: : )
